stress_test/emulator_test_basic2.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2016 Intracom S.A. Telecom Solutions. All rights reserved.
```

Comment:
be explicit, rename this file to sth having "multinet" so that we can tell this is a multinet-based test